### ISSUE TITLE
Bad site rendering in Firefox and Edge

### ISSUE THREAD

**Issue by bdowning** [Turfjs/turf-www#132]
There appears to be some sort of WebKit- or Blink-specific CSS at play in the new documentation site, as the site is misrendered in other engines.  I tried to figure out what the CSS is doing but couldn't make much sense of it at a brief glance.  It seems quite a bit more complex than it needs to be for a simple side-by-side layout, however.

Sorry that I don't have time right now to try to fix this.

Firefox (Nightly, but it happens in the released version too):
![2017-12-06-133340_1128x1076_scrot](https://user-images.githubusercontent.com/366426/33681688-e2709974-da8a-11e7-85b6-d9b706d6d8b6.png)
![2017-12-06-133351_1128x1076_scrot](https://user-images.githubusercontent.com/366426/33681692-e6b8ea68-da8a-11e7-9027-fd658f82a900.png)

Edge:
![2017-12-06-133705_1023x894_scrot](https://user-images.githubusercontent.com/366426/33681698-e9853ca6-da8a-11e7-9f9f-f972a9cd45ec.png)

**Comment by rowanwins** [comment]
Hi @bdowning 

Yep the issue also got reported in the main turf repo. 

This resolves the issue (at least in IE)
````
//replace 
.sideCol {
    min-height: filll;
}
//with
.sideCol {
   min-height: 100vh;
}
````
Not sure if @DenisCarriere can take a look quickly otherwise I'll get it done tonight.

PS The layout is a smidge more complex than it needs to be because of the fixed top bit in the sidebar.

**Comment by bdowning** [comment]
I can confirm that changing .sideCol's min-height to 100vh in the devtools appears to fix the rendering in Firefox as well.  Sorry I didn't check the main repo before reporting this here.

**Comment by rowanwins** [comment]
All good, always better to safe than sorry!

**Comment by DenisCarriere** [comment]
@rowanwins If that's the only requirement to fix it I can have a look at it now, I'll download Firefox to make sure it works before I publish

**Comment by rowanwins** [comment]
Resolved by #133
